CGBEmu/source/App.cpp

```diff
@@ -100,7 +125,67 @@ void App::PrintRendererInfo() const
 void App::PrintFramerate() const
 {
 	const static uint64_t freq = SDL_GetPerformanceFrequency();
-	const double seconds = (m_FrameEndTime - m_FrameStartTime) / static_cast< double >(freq);
+	const double seconds = (m_FrameEndTime - m_FrameStartTime) / static_cast<double>(freq);
 	const int fps = static_cast<int>(1.0 / seconds);
-	std::cout << "Frame time: " << seconds * 1000.0 << "ms FPS: " << fps << std::endl;
+	//std::cout << "Frame time: " << seconds * 1000.0 << "ms FPS: " << fps << std::endl;
+}
+
+void App::moverplayers(unsigned int h)
+{
+	SDL_Event event;
+	SDL_UpdateTexture(m_Texture, NULL, m_ScreenBuffer.data(), WindowWidth * PixelByteSize);
```

Comment:
Move the SDL_* functions out from the MovePlayer function and put them at the end of the `Update` function